### ISSUE TITLE
Web searches to use specific profiles

### ISSUE THREAD

**Issue by j-a-s-o-n-g** [Flow-Launcher/Flow.Launcher#2653]
So I have two chrome profiles, let's call it "Main" and "Work". If I trigger the Web Searches addon while my "Work" chrome profile is open, it will automatically use that profile for said search. However I would like to strictly open any searches in my "Main" chrome profile, no matter the focus. This is the command to open Chrome in a certain profile:

`"C:\Program Files\Google\Chrome\Application\chrome.exe" --profile-directory="Default"` (Replace Default with any named profile)

Tl;DR: if the default browser is Chrome, there should be an [optional] option to choose a specific profile to open all requests in.

Thanks!

**Comment by ArkitectNaut** [comment]
It would be great if this could also work for Firefox

**Comment by DavidGBrett** [comment]
I've opened a PR which should allow users to implement this themselves by adding any extra arguments they want.
